File: Data Structures & Algorithms/anagram-groups/submission-0.cpp
class Solution {
public:
    vector<vector<string>> groupAnagrams(vector<string>& strs) {
        unordered_map<string,vector<string>>groups;
        vector<vector<string>>gStrings;

        for(string& s : strs){
            string key = s;
            sort(key.begin(),key.end());
            groups[key].push_back(s);
        }
        for(auto& entries : groups){
            gStrings.push_back(entries.second);
        }
        return gStrings;
    }
};

File: Data Structures & Algorithms/is-anagram/submission-0.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        sL= list(s)
        tL= list(t)
        sL.sort()
        tL.sort()
        if len(s) != len(t):
            return False
        if sL == tL:
            return True
        return False
        

        